dll/win32/shell32/shlexec.cpp

```diff
@@ -802,6 +802,15 @@ static UINT SHELL_FindExecutable(LPCWSTR lpPath, LPCWSTR lpFile, LPCWSTR lpVerb,
         xlpFile[0] = '\0';
     }
 
+    /* Clear any trailing periods */
+    for (SIZE_T i = wcslen(xlpFile); i > 0; i--)
+    {
+        if (i > 0 && xlpFile[i - 1] == '.')
+            xlpFile[i - 1] = 0;
```

Comment:
Consistency with line 802
```suggestion
            xlpFile[i - 1] = '\0';
```